src/templates/post.js

```diff
@@ -1,28 +1,28 @@
-import React, { Component } from 'react'
-import Img from 'gatsby-image'
+import React from 'react'
+import Introduction from './components/introduction'
+import ImageGrid from './components/image-grid'
+import StackedImages from './components/full-width-image'
 
-class PostTemplate extends Component {
-  render() {
-    const post = this.props.data.wordpressPost
+const PostTemplate = ({ data }) => (
```

Comment:
We should be passing in `wordpressPost` here instead of `data` to avoid duplication and make this component less fragile